Code/CryGame/Actors/Player/NanoSuit.cpp

```diff
@@ -691,13 +691,13 @@ void CNanoSuit::Hit(int damage)
 		pMaterialEffects->ExecuteEffect(id, params);
 	}
 
-	/*if(damage > 10.0f && m_pOwner && m_pOwner->GetHealth() > 0)
+	if(damage > 10.0f && m_pOwner && m_pOwner->GetHealth() > 0)
```

Comment:
I would add a cvar for it because it changes the vanilla behavior. Something like `mp_suitHitReaction`.